amethyst_ui/src/lib.rs

```diff
@@ -22,17 +22,19 @@ extern crate unicode_segmentation;
 extern crate winit;
```

Comment:
#[macro_use]
extern crate shred_derive;

Reply:
Done. I needed to add shred as well to make the compiler be quiet.